Okta/OktaApi.swift

```diff
@@ -11,27 +11,53 @@
  */
 
 open class OktaApi: NSObject {
-    class func post(_ url: URL, headers: [String: String]?, postData: String?, callback: @escaping ([String: Any]?, OktaError?) -> Void) {
+    class func post(_ url: URL,
+                    headers: [String: String]?,
+                    postData: String?,
+                    callback: @escaping ([String: Any]?, OktaError?) -> Void) {
+
         // Generic POST API wrapper
-        
         var request = URLRequest(url: url)
         request.httpMethod = "POST"
-        request.allHTTPHeaderFields = headers
-        
+        request.allHTTPHeaderFields = headers != nil ? headers : request.allHTTPHeaderFields
+        request.addValue(
+            "okta-sdk-appauth-ios/\(VERSION) iOS/\(UIDevice.current.systemVersion)",
```

Comment:
Is there any way to get device info as well as the iOS version?  in the SDK's we have `sdk-name/version language/version os/version`  would it be useful to add to the end of that something like `iPhoneX/blah` if it is available?

Reply:
Done `okta-sdk-appauth-ios/0.3.0 iOS/11.2 Device/x86_64` (Indicates a simulator)